portfolio/src/main/webapp/script.js

```diff
@@ -12,13 +12,36 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
+
+
 /**
- * Fetches greeting from the server and adds it to the DOM
+ * Fetches comments from the server and adds it to the DOM
  */
-function getData() {
-    fetch("/data").then(response => response.text()).then((data) => {document.getElementById("data").innerHTML = data});
+function getComments() {
+    fetch("/data").then(response => response.json()).then((commentsData) => {
+        const commentsContainer = document.getElementById("comments-container");
+        commentsContainer.innerHTML = '';
+        for (var i = 0; i < commentsData.length; i++) {
+            var commentData = commentsData[i];
```

Comment:
Best practices is to avoid using var: https://google.github.io/styleguide/jsguide.html#features-use-const-and-let

Reply:
I have fixed this in now! Thanks!